spec/classes/agent_spec.rb

```diff
@@ -56,7 +56,7 @@
         end
       end
       ['service','cron','only_service'].each do |agent_method|
-        manage_repos = false if facthash['osfamily'] == 'FreeBSD'
+        manage_repos = false if facthash['osfamily'] == 'FreeBSD' || true
```

Comment:
I don't understand the ` || true` here -- wouldn't this be equivalent to removing the `if facthash['osfamily'] == 'FreeBSD'` entirely?